blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -75,15 +75,15 @@ export class Activity extends Component {
         const head = 'pipelinedetail.activity.header';
 
         const status = t(`${head}.status`, { defaultValue: 'Status' });
-        const build = t(`${head}.build`, { defaultValue: 'Build' });
+        const run = t(`${head}.run`, { defaultValue: 'Run' });
```

Comment:
Yes that is for translation only and it is the last fallback in case all i18n  services are down. You would see the `defaultValue` which by definition should be the same as in the english i18n locale.